docs/network/index.md

```diff
@@ -21,22 +21,22 @@ Overview of Celo Mainnet, Alfajores Testnet and Baklava Testnet.
 | Network Status     | [https://stats.celo.org](https://stats.celo.org)                                                                     |
 | Validator Explorer | [https://validators.celo.org](https://validators.celo.org)                                                           |
 
-## Celo Alfajores
+## Celo Alfajores L2
 
 | Name            | Value                                                                                                                                           |
 | --------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- |
 | Network Name    | Celo Alfajores                                                                                                                                  |
 | Description     | The Developer Testnet network                                                                                                                   |
 | Currency Symbol | CELO                                                                                                                                            |
 | Chain ID        | 44787                                                                                                                                           |
-| RPC Endpoint (best effort)   | [https://alfajores-forno.celo-testnet.org](https://alfajores-forno.celo-testnet.org) |
-| RPC Nodes       | [https://docs.celo.org/network/node/forno#alfajores-testnet](https://docs.celo.org/network/node/forno#alfajores-testnet)                        |
-| Block Explorers  | <ul><li>[https://explorer.celo.org/alfajores](https://explorer.celo.org/alfajores)</li><li>[https://alfajores.celoscan.io](https://alfajores.celoscan.io)</li></ul>  |
+| RPC Endpoint (best effort)   | [https://forno.alfajores.celo-testnet.org/](https://forno.alfajores.celo-testnet.org/) |
+| RPC Nodes       | <ul><li>Ethereum JSON-RPC endpoint: [https://forno.alfajores.celo-testnet.org/](https://alfajores-forno.celo-testnet.org) (op-geth kind)</li><li>OP RPC endpoint: [https://op.alfajores.celo-testnet.org/](https://op.alfajores.celo-testnet.org/) (op-node kind)</li></ul>                      |
```

Comment:
1. Are these the correct Forno URLs?
2. I would avoid only pointint to Forno, ideally we want builders to use more endpoints and not direct all traffic to Forno.

Reply:
These are the ones in the Celo L2 FAQ. Do we have more nodes? I wasn't aware of any besides forno

Reply:
@sodofi , you can add other public available RPC endpoints, like Ankr, and recommend updated (to L2) RPCs Quicknode and Infura